orangecontrib/spectroscopy/preprocess/utils.py

```diff
@@ -215,3 +215,20 @@ def interp1d_wo_unknowns_scipy(x, ys, points, kind="linear"):
 def edge_baseline(x, y):
     i = np.array([0, -1])
     return interp1d(x[i], y[:, i], axis=1)(x) if len(x) else 0
+
+
+def replace_infs(array):
+    """ Replaces inf and -inf with nan """
+    array[np.isinf(array)] = np.nan
+    return array
+
+def divide_no_inf(numerator, denominator, in_place=False):
```

Comment:
You added this function to suppress the warning, right? If warning suppression was not needed, I would prefer to call replace_infs directly.

Reply:
Actually, at first there was no replace_infs(), but then I noticed np.log10() with zeros in the reference caused the same problem (thanks to the test). 

I'll just simplify this and remove divide_no_inf entirely.